tools/fork-network/src/cli.rs

```diff
@@ -160,16 +129,36 @@ impl ForkNetworkCommand {
         let runtime_config_store = RuntimeConfigStore::new(None);
         let runtime_config = runtime_config_store.get_config(PROTOCOL_VERSION);
         let storage_bytes = runtime_config.fees.storage_usage_config.num_bytes_account;
+
+        let mut new_validator_accounts = vec![];
+
         let liquid_balance = 100_000_000 * NEAR_BASE;
-        storage_mutator.set_account(
-            self_validator.validator_id().clone(),
-            Account::new(liquid_balance, self_account.amount, CryptoHash::default(), storage_bytes),
-        )?;
-        storage_mutator.set_access_key(
-            self_account.account_id.clone(),
-            self_account.public_key.clone(),
-            AccessKey::full_access(),
-        )?;
+        let new_validators: Vec<Validator> = serde_json::from_reader(BufReader::new(
+            File::open(home_dir.join(self.validators)).expect("Failed to open validators JSON"),
+        ))
+        .expect("Failed to read validators JSON");
+        for validator in new_validators.into_iter() {
+            let validator_account = AccountInfo {
+                account_id: validator.account_id,
+                amount: 50_000 * NEAR_BASE,
```

Comment:
Any reason not allow specifying, possibly optionally the amount of tokens in the configuration as well?